pkg/manager/manager.go

```diff
@@ -0,0 +1,9 @@
+package manager
+
+import "context"
+
+type Interface interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Interface should have comment or be unexported